scalastyle-config.xml

```diff
@@ -115,11 +115,6 @@
  </check>
  <check level="error" class="org.scalastyle.scalariform.DeprecatedJavaChecker" enabled="true"/>
  <check level="error" class="org.scalastyle.scalariform.EmptyClassChecker" enabled="true"/>
- <check level="error" class="org.scalastyle.scalariform.ClassTypeParameterChecker" enabled="true">
```

Comment:
The way scalacheck does them breaks some of the common idioms, e.g.:

sealed trait X
...
assume 'y' is of type X
```
y match {
   case SomeSpecific(x) =>
   case SomeOtherSpecific(z) =>
   case catchOthers => 
}
```
catchOthers is _guaranteed_ to be of type 'X' and the scalastyle check complains because the type isn't there.